src/streams/StreamItem.js

```diff
@@ -91,6 +91,7 @@ export default class StreamItem extends PureComponent<Props> {
       showSwitch,
       isSwitchedOn,
       unreadCount,
+      accessibilityLabel,
```

Comment:
Similarly to my comment on `TopicItem`, let's not add an `accessibilityLabel` to `StreamItem`. Instead, let's have `StreamItem` build `accessibilityLabel` itself, since it already has the data it needs, in other props.

I see one of `StreamItem`'s callers doesn't provide an `unreadCount`. That's fine; `StreamItem` can decide to do one thing if it's passed an `unreadCount`, and a different thing if it's not.